src/session-pool.ts

```diff
@@ -164,6 +169,7 @@ export interface SessionPoolOptions {
   writes?: number;
   incStep?: number;
   databaseRole?: string | null;
+  observabilityOptions?: ObservabilityOptions;
```

Comment:
We should not add `observabilityOptions` as part of `SessionPoolOptions` . Customer pass the SessionPoolOptions and it will be confusing for them on what to use. Create this as a private property in sessionpool class, similar to _traces property

Reply:
Alright removed, thanks @surbhigarg92!